examples/rust/blackboard_event_based_communication/creator.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2025 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use core::time::Duration;
+use iceoryx2::prelude::*;
+
+const CYCLE_TIME: Duration = Duration::from_secs(1);
+
+fn main() -> Result<(), Box<dyn core::error::Error>> {
+    set_log_level_from_env_or(LogLevel::Trace);
+    let node = NodeBuilder::new().create::<ipc::Service>()?;
+    let key = 99;
+
+    let service = node
+        .service_builder(&"My/Funk/ServiceName".try_into()?)
+        .blackboard_creator::<u32>()
+        .add::<u64>(key, 2023)
+        .create()?;
+
+    let event_service = node
+        .service_builder(&"My/Funk/ServiceName".try_into()?)
+        .event()
+        .open_or_create()?;
+    let listener = event_service.listener_builder().create()?;
+
+    let reader = service.reader_builder().create()?;
```

Comment:
I know it is just for demonstration and not yet the final example but I would move the reader into the opener and the writer into the creator. Otherwise, it somehow is counter-intuitive and I start to wonder why it is the other way around.

But this can be done in the blackboard example pr

Reply:
I'll change this in the follow-up.